CanvasPlusPlayground/Features/People/PeopleView.swift

```diff
@@ -61,12 +89,20 @@ struct PeopleView: View {
     }
 
     private var displayedUsers: [User] {
-        searchText.isEmpty ?
-        peopleManager.users :
-        peopleManager.users
-            .filter {
+        var result = peopleManager.users
+        if !searchText.isEmpty {
+            result = result.filter {
                 $0.name?.localizedCaseInsensitiveContains(searchText) ?? true
             }
+        }
+
+        selectedTokens.forEach { token in
+            result = result.filter {
+                $0.role?.contains(token.text) ?? true
+            }
+        }
```

Comment:
Small detail, but perhaps we can combine these with a `compactMap`. The number of people can be really large, so it would be better to avoid iterating twice.

Reply:
Updated with a single `filter` and `allSatisfy` for the tokens